src/e3/fs.py

```diff
@@ -549,6 +549,9 @@ def splitall(path: str) -> tuple[str, ...]:
     ".gitreview",
     ".mailmap",
     ".idea",
+    ".python-version",
```

Comment:
What about `.flake8`. It does not contain any important data, but does it belong to the package ?

Reply:
`.github` could also be added to the list.

Reply:
@grouigrokon For .flake8 I don't think so, we provide the sources, for me that's part of it.

.github added.

Sorry for the delay, this PR was totally off my radar.